plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -93,19 +94,23 @@ public class IcebergFileWriterFactory
     private final FileFormatDataSourceStats readStats;
     private final OrcWriterStats orcWriterStats = new OrcWriterStats();
     private final OrcWriterOptions orcWriterOptions;
+    private final DateTimeZone parquetTimeZone;
```

Comment:
Is the parquetTimeZone relevant for the writer as well in Iceberg?
Shouldn't UTC be enough?

Reply:
I thought that it should be consistent.  I applied the property value (iceberg.parquet.time-zone) to both Parquet's reader and Parquet's writer in Iceberg.